node/pkg/db/governor.go

```diff
@@ -152,19 +211,44 @@ func (d *Database) GetChainGovernorData(logger *zap.Logger) (transfers []*Transf
 			}
 
 			if IsPendingMsg(key) {
-				msg, err := common.UnmarshalMessagePublication(val)
+				p, err := UnmarshalPendingTransfer(val)
 				if err != nil {
 					return err
 				}
 
-				pending = append(pending, msg)
+				pending = append(pending, p)
 			} else if IsTransfer(key) {
 				v, err := UnmarshalTransfer(val)
 				if err != nil {
 					return err
 				}
 
 				transfers = append(transfers, v)
+			} else if isOldPendingMsg(key) {
+				msg, err := common.UnmarshalMessagePublication(val)
+				if err != nil {
+					return err
+				}
+
+				p := &PendingTransfer{ReleaseTime: msg.Timestamp.Add(time.Duration(time.Hour * 72)), Msg: *msg}
```

Comment:
I'm wondering if we should consider not trusting msg.Timestamp here and just override it with now, similar to how we would handle new PENDING2 style messages.  It could guard against backdated PENDING1 messages prior to upgrade/migration and prevent them from having instant finality post migration to PENDING2.